pages/index.js

```diff
@@ -55,14 +65,18 @@ export default function HomePage() {
         <h2>Anstehende Aufgaben:</h2>
         {submittedData.length > 0 && (
           <div className="submitted-data">
-            {submittedData.map((data, index) => (
-              <p key={index}>
-                Was ist zu tun: {data.option} <br />
-                Bis wann erledigt: {data.date}
-              </p>
+            {submittedData.map((data) => (
+              <div key={data.id} className="task-item">
```

Comment:
Instead of using css classes you should work with styled components. :)

Reply:
Yes that´s right. I will use style components in the future, when my app works clear with all the functions :)